fractions/wildfly/elytron/src/main/java/org/wildfly/swarm/elytron/ElytronFraction.java

```diff
@@ -133,40 +135,48 @@ public ElytronFraction applyDefaults() {
             realm.className("org.wildfly.swarm.elytron.runtime.Realm");
```

Comment:
We can't use `.properties`-based realms, but `org.wildfly.swarm.elytron.runtime.Realm` is essentially an empty stub. Is there a way to do `.properties`-like realms, backed by YAML configuration, like we have in legacy security domains?